components/sync15/src/telemetry.rs

```diff
@@ -9,13 +9,13 @@ use std::collections::HashMap;
 use std::time;
 
 // We use serde to serialize to json (but we never need to deserialize)
```

Comment:
micro-nit - this comment was kinda explaining we only `serde::ser` was imported, but now it's moved away from that import, so makes even less sense than it did before :) I suggest you just remove this comment.